### ISSUE TITLE
Lexical editor config is discregarded when used within a Block

### ISSUE THREAD

**Issue by PavelGolodoniuc** [payloadcms/payload#8303]
### Describe the Bug

If you have a Block that is allowed within a Lexical editor that itself has a `richText` field with a Lexical editor, that block's lexical editor always uses default configuration regardless of what you provide it.

### To Reproduce

```TypeScript
export const Columns: Block = {
	slug: "columns",
	labels: {
		singular: "Columns",
		plural: "Columns",
	},
	fields: [
		{
			name: "style",
			type: "select",
			options: [
				{
					label: "Two-column split (6:6)",
					value: "dual-6-6",
				},
				{
					label: "Two-column split (5:1:6)",
					value: "dual-5-1-6",
				},
				{
					label: "Two-column split (4:1:7)",
					value: "dual-4-1-7",
				},
				{
					label: "Two-column split (7:1:4)",
					value: "dual-7-1-4",
				},
			],
		},
		{
			name: "colOne",
			label: "First column content",
			type: "richText",
			editor: lexicalEditor(defaultConfig),     // <-------------- PROBLEM IS HERE
		},
		{
			name: "colTwo",
			label: "Second column content",
			type: "richText",
			editor: lexicalEditor(defaultConfig),     // <-------------- PROBLEM IS HERE
			admin: {
				condition: (_data: any, siblingData: any) => !(<string>siblingData?.style).startsWith("single"),
			},
		},
	],
};
```

In the example above, both editors, defined as `editor: lexicalEditor(defaultConfig)` completely disregard `defaultConfig`.

### Payload Version

2.28.0

### Adapters and Plugins

"@payloadcms/richtext-lexical": "^0.11.3"

**Comment by zeitchef** [comment]
I'm facing a similar problem on v3 beta.107 - crashes the admin backend with the following error:
<img width="826" alt="Screenshot 2024-09-23 at 13 43 57" src="https://github.com/user-attachments/assets/3f019db4-19f5-4c23-b82c-2657d416140a">
I've already created a separate issue.

**Comment by AlessioGr** [comment]
We currently do not have the capacity to fix this issue in 2.0 and are focusing on security and critical issues for 2.0. If this is still an issue in payload 3.0, please open a separate issue. There have been tons of improvements in regards to blocks & lexical, and the issue is likely already fixed there :)

This is likely unrelated to https://github.com/payloadcms/payload/issues/8274, as 2.0 does not have import maps
